Makefile

```diff
@@ -75,71 +75,74 @@ KIND:=$(DEPENDENCIES)/kind/$(KIND_VERSION)
 $(KIND):
 	@echo "installing kind $(KIND_VERSION)..."
 	$(eval KIND_TMP := $(shell mktemp -d))
-	@(cd "$(KIND_TMP)" \
-		&& go mod init tmp \
-		&& go get "sigs.k8s.io/kind@$(KIND_VERSION)" \
+	@(cd "$(KIND_TMP)"; \
+		go mod init tmp; \
+		go get "sigs.k8s.io/kind@$(KIND_VERSION)"; \
 	) 2>&1 | sed 's/^/  /'
-	@rm -rf "$(KIND_TMP)" "$(dir $(KIND))" \
-		&& mkdir -p "$(dir $(KIND))" \
-		&& touch "$(KIND)" \
-		&& echo
+	@rm -rf "$(KIND_TMP)" "$(dir $(KIND))"; \
+		mkdir -p "$(dir $(KIND))"; \
+		touch "$(KIND)"; \
+		echo;
```

Comment:
I think these can be written as separate commands (ie, lines not joined by `; \`) because of 

```
SHELL=/bin/bash
.SHELLFLAGS=-euo pipefail -c
```

bash will bubble up an eventual error to make and then make will stop execution of the target recipe when it encounters an error